app/views/mw_interactives/edit.html.haml

```diff
@@ -15,6 +15,12 @@
     .clear
     This width and height will be used to maintain a fixed aspect ratio for the interactive.
   %br
+  = field_set_tag 'Pre-flight URL' do
+    = f.text_area :preflight_url, size: "45x4"
+  %div{:style => "margin-bottom:5px;"}
+    %em Warning: A Pre-flight URL automatically enables Click to Play.
+    Please provide an image_url to use a Pre-flight URL.
```

Comment:
When I was testing it out, it actually looks fine (for CODAP at least) if there's not an image_url specified. You just end up with the words "Click here to start the interactive" over a white space.

If we end up using the flow @scytacki described, then this will get removed from the authoring anyway.